CDCgov/NEDSS-Modernization
# Cnft1 749 skip link context hook

## Description

Adding a context for the Skip Link, this is a invisible link that should wrap up the app and be the first tabbing option for the user 
The goal of this skip link is to navigate the user to main content of the page.

Note: There is an example of the usage of the skip link in advanced search for testing purposes
Proper usage of main content will be determined later

## Tickets

https://cdc-nbs.atlassian.net/browse/CNFT1-749

## Steps to verify

1.Login
2. Click on tab button
3. Observe the skip link focused
4. navigate to main content
